src/components/carousel/index.jsx

```diff
@@ -29,6 +29,28 @@ const DefaultPreviousButton = ({ id, onClick }) => (
 	</Button>
 );
 
+const DefaultAdditionalPreviousButton = ({ id, onClick }) => (
+	<Button
+		id={id}
+		onClick={onClick}
+		label="Previous Slide"
+		className={`${COMPONENT_CLASS_NAME}__button ${COMPONENT_CLASS_NAME}__button--additional-previous`}
+	>
+		{"<"}
```

Comment:
Can use use an `<Icon />` here? `ChevronRight`

Reply:
sure. importing one component into the other ok?

Reply:
Good question - we've not doing it yet - maybe we use an HTML entity then rather than the text >

Reply:
@matthewroach updated
![Screen Shot 2022-04-20 at 16 20 04](https://user-images.githubusercontent.com/5950956/164324908-caf6e35c-be9e-4fe8-8f9a-5feb89d5785e.png)

Reply:
@matthewroach updated